README.windows.md

```diff
@@ -178,6 +180,8 @@ Julia can be also compiled from source in [Cygwin](http://www.cygwin.com), using
     make -j 4   # Adjust the number of cores (4) to match your build environment.
 ```
 
+  4.  See the steps in the MSYS2 section on building windows extras if needed.  
```

Comment:
trailing whitespace here too

The `win-extras` are technically needed whether you build from Cygwin or MSYS2.